go/vt/discovery/healthcheck.go

```diff
@@ -228,6 +229,9 @@ type HealthCheck interface {
 
 	// Unsubscribe removes a listener.
 	Unsubscribe(c chan *TabletHealth)
+
+	// LoadTabletsTrigger returns a channel that is used to inform when to load tablets.
+	LoadTabletsTrigger() chan struct{}
```

Comment:
IMO, `GetLoadTabletsTrigger` is a better name.

Reply:
It makes perfectly sense.
Changed by [a710157](https://github.com/vitessio/vitess/pull/12893/commits/a7101571a76869bb0fb0f248553ed1b4a9235630).